src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,16 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+public class ListCommand extends Command {
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Should there be a javadoc comment for this public method? This is also seen in other instances in the code.